networkit/_NetworKit.pyx

```diff
@@ -54,6 +53,60 @@ import numpy as np
 # currently part of the Cython distribution).
 cimport numpy as np
 cimport cython
+from libc.stdlib cimport free
+from cpython cimport PyObject, Py_INCREF
+# Numpy must be initialized. When using numpy from C or Cython you must
+# _always_ do that, or you will have segfaults
+np.import_array()
+
+cdef class ArrayWrapper:
```

Comment:
Can you say a bit about why we need to implement this ourselves? Looks like a standard thing that should exist somehow.

Reply:
Yes, I thought so as well, but it seems that this boilerplate is still necessary to wrap a numpy array around a c pointer.